### ISSUE TITLE
windows-latest jdk-17 unit tests always failed 

### ISSUE THREAD

**Issue by doofin** [scalameta/metals#6110]
### Describe the bug

CI windows-latest jdk-17 unit tests always failed, should we remove it? 

### Operating system

None

### Editor/Extension

None

### Version of Metals

v1.2.1

**Comment by kasiaMarek** [comment]
I don't think we should remove it. The failing suite `DebugProtocolSuite` is flaky and should probably be marked as such, so it doesn't stop the CI from passing.

**Comment by tgodzik** [comment]
Ideally, I wouldn't ignore the whole suite, best to figure out what the issue is and do a fix or a workaround.
